next.config.js

```diff
@@ -10,6 +10,10 @@ const nextConfig = {
   reactStrictMode: false,
 
   pageExtensions: ["js", "jsx", "mdx", "ts", "tsx"],
+  env: {
+    NETWORK: process.env.NETWORK,
+    DAPP_DEFINITION_ADDRESS: process.env.DAPP_DEFINITION_ADDRESS,
+  },
```

Comment:
Do we really need this doubling? next should be able to pick up the env vars by default

Reply:
Ran into an issue that they were undefined in browser. I followed a Stackoverflow said that you had to do this for browser/client side. Looks like the new docs suggest another strategy.
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

Reply:
maybe using `NEXT_PUBLIC_` prefix for the env vars?